src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Learn more about Recurring Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Create Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Collect Sustaining Gifts with Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Individual in GS for End User</shortDescription>
+        <value>For most nonprofits, donations come from individual contributions. Track individual donations by creating an Opportunity from a Contact record.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Donation Individual in GS for End User</shortDescription>
+        <value>Learn more about Individual Donations in Trailhead</value>
+    </labels>
+     <labels>
+    <fullName>gseuChecklistItemDonationIndividualPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Individual in GS for End User</shortDescription>
+        <value>Enter Individual Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Individual in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Title of Individual Donation checklist item in Get Started for End Users.